src/components/organisms/AdminVenueView/components/RunTabToolbar/RunTabToolbar.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { faPaperPlane } from "@fortawesome/free-solid-svg-icons";
+
+import { makeUpdateBanner } from "api/bannerAdmin";
+
+import { venueInsideUrl } from "utils/url";
+
+import { InputField } from "components/atoms/InputField";
+import { ButtonNG } from "components/atoms/ButtonNG/ButtonNG";
+
+import "./RunTabToolbar.scss";
+import { useAsyncFn } from "react-use";
+
+export interface RunTabToolbarProps {
+  venueId?: string;
+}
+
+const noop = () => undefined;
+
+export const RunTabToolbar: React.FC<RunTabToolbarProps> = ({ venueId }) => {
+  const { register, getValues } = useForm<{
+    message: string;
+  }>({
+    mode: "onSubmit",
+  });
+
+  const [{ loading: isUpdatingBanner }, updateBanner] = useAsyncFn(async () => {
+    if (!venueId) return;
+    const bannerMessage = getValues().message;
+    await makeUpdateBanner(venueId, noop)(bannerMessage);
+  }, [getValues, venueId]);
+
+  return (
+    <div className="RunTabToolbar__wrapper">
+      <form
+        className="RunTabToolbar__toolbar RunTabToolbar__toolbar--left RunTabToolbar__form"
+        onSubmit={noop}
```

Comment:
can we get rid of this noop here? I assume that's done because there's somewhere event propagation going, we need to redo the element cascading, so that it's eliminated

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1705/commits/3ca884136419f135a2893c7e0fc2ee4af1d3301b